### PR TITLE
[ Fabric ] New Org is added to existing consortium and to the existing channel

### PR DESCRIPTION
**Pull request by alvaropicazo**
Signed-off-by: alvaropicazo <alvaro.picazo.haase@accenture.com>

**Changelog**
- Added new setup/genesis folder where the genesis block is updated by the orderer using syschannel to add the org to the existing consortium
- Fixed the creation of a new anchorpeer for the new org 
- Updated some files related to the sign and update of the block to have that org in the desired channel

 

**Reviewed by**
@jagpreetsinghsasan 
@suvajit-sarkar 

 **Screenshots**
In this screenshot, we can see that the existing creator org (calv) and the new org (malv) are in the same consortium
consortium

In this screenshot, we can see that both orgs have one anchorpeer each, the existing one and the new one created.
anchor_added_to_channel

**Linked issue**
#1297

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml [hunk 1]
```diff
@@ -44,4 +44,7 @@
     GIT_BRANCH: "{{ org.gitops.branch }}"
     GIT_RESET_PATH: "platforms/hyperledger-fabric/configuration"
     msg: "[ci skip] Pushing CLI value files"
-  when: peer.cli is defined and peer.cli == "enabled" 
+  loop: "{{ peers }}"
+  loop_control:
+    loop_var: peer    
+  when: peer.cli is defined and peer.cli == "enabled"
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml:50]: Newline

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/create_block/tasks/main.yaml [hunk 2]
```diff
@@ -24,10 +24,11 @@
...
     component_name: "{{ participant.name | lower }}"
+    orderer_address: " {{ participant.ordererAddress }} "
     os: "{{ fabric.os }}"
     arch: "{{ fabric.arch }}"
     version: "{{ network.version }}"
   loop: "{{ participants }}"
   loop_control:
     loop_var: participant
-  when: participant.org_status == 'new'
+  when: participant.org_status == 'new'
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/create_block/tasks/main.yaml:34]: Newline

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/syschannel_block/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  vars:
+    orderer: "{{ org.services.orderers | first }}"
+    new_org_query: "participants[?org_status=='new']"
+    channel: "{{ chann }}"
+    new_org: "{{ chann | json_query(new_org_query) | first }}"
+  loop: "{{ network.channels }}"
+  loop_control:
+    loop_var: chann
+  when: new_org != ""  
+
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/syschannel_block/tasks/main.yaml:29]: Newline.

File: platforms/hyperledger-fabric/configuration/roles/create/new_organization/syschannel_block/tasks/nested_create_cli.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    ORDERER_CA: "/opt/gopath/src/github.com/hyperledger/fabric/crypto/orderer/tls/ca.crt"
+  vars: 
+    kubernetes: "{{ org.k8s }}"
+    ordererAddress: "{{ orderer.name | lower }}.{{ org.external_url_suffix }}:8443"
+
+# Delete the orderer cli   
+- name: "delete cli {{ orderer.name }}-{{ org.name }}-cli"
+  shell: |
+    KUBECONFIG={{ org.k8s.config_file }} helm uninstall {{ orderer.name }}-{{ org.name }}-cli
+
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-fabric/configuration/roles/create/new_organization/syschannel_block/tasks/nested_create_cli.yaml:73]: Newline. This is a `tab` + `newline`

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

**Comment by jagpreetsinghsasan** [comment]
Also make sure, to squash the commits in 1 commit.

**Review by jagpreetsinghsasan — APPROVED** [review]

**Review by sownak — APPROVED** [review]

**Pull request merged**